sw/device/silicon_creator/manuf/base/perso_tlv_data.h

```diff
@@ -121,6 +121,10 @@ typedef struct perso_tlv_cert_obj {
    * LTV object size (in bytes).
    */
   size_t obj_size;
+  /**
+   * LTV object type.
```

Comment:
nit: TLV object type.

Reply:
Need to align comments across several files, will do in follow up if that is ok?

Reply:
SGTM